### ISSUE TITLE
Uncaught Error: No protocol method Evaluator.evaluate defined for type null: 

### ISSUE THREAD

**Issue by aviflax** [mauricioszabo/atom-chlorine#179]
### Steps to reproduce

1. I saved the open file, triggering Chlorine to re-load it. (Using simple JVM Clojure reloading.)
2. A clojure.lang.ExceptionInfo object was printed to my console, in red.
3. I tried to expand the object; then this _Uncaught Error_ alert popped up

### Environment

**Atom**: 1.46.0 x64
**Electron**: 4.2.7
**OS**: Mac OS X 10.15.4
**Thrown From**: [chlorine](https://github.com/mauricioszabo/atom-chlorine) package 0.6.1

### Stack Trace

Uncaught Error: No protocol method Evaluator.evaluate defined for type null: 

```
At /Users/avi.flax/.atom/packages/chlorine/lib/cmds.js:61

Error: No protocol method Evaluator.evaluate defined for type null: 
    at Cb (/packages/chlorine/lib/cmds.js:349:291)
    at vY (/packages/chlorine/lib/cmds.js:1629:220)
    at Function.c (/packages/chlorine/lib/cmds.js:1632:135)
    at /packages/chlorine/lib/cmds.js:2316:198
    at Object.t (/packages/chlorine/lib/cmds.js:40:444)
    at /packages/chlorine/lib/cmds.js:41:28)
    at /packages/chlorine/lib/cmds.js:41:80)
    at /packages/chlorine/lib/cmds.js:41:252)
    at mi (/packages/chlorine/lib/cmds.js:61:54)
    at ae (/packages/chlorine/lib/cmds.js:60:382)
    at Pc (/packages/chlorine/lib/cmds.js:61:217)
    at Xf (/packages/chlorine/lib/cmds.js:63:389)
    at Ff (/packages/chlorine/lib/cmds.js:263:426)
    at /packages/chlorine/lib/cmds.js:43:423)
    at fe (/packages/chlorine/lib/cmds.js:71:207)
    at Sc (/packages/chlorine/lib/cmds.js:70:272)
    at l.unstable_runWithPriority (/packages/chlorine/lib/cmds.js:37:101)
    at cb (/packages/chlorine/lib/cmds.js:89:333)
    at /packages/chlorine/lib/cmds.js:263:286)
    at ui (/packages/chlorine/lib/cmds.js:70:87)
```

### Commands

```
     -1:04.4.0 core:move-to-bottom (input.hidden-input)
  4x -1:04.1.0 core:move-up (input.hidden-input)
     -1:03.3.0 core:move-down (input.hidden-input)
     -1:02.8.0 editor:move-to-end-of-screen-line (input.hidden-input)
  2x -1:00.6.0 pane:show-next-item (input.hidden-input)
     -0:59.4.0 pane:show-previous-item (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-light-syntax.theme-one-light-ui)
     -0:59.2.0 core:close (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-light-syntax.theme-one-light-ui)
     -0:58.7.0 pane:show-next-item (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-light-syntax.theme-one-light-ui)
  2x -0:55.8.0 pane:show-previous-item (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-light-syntax.theme-one-light-ui)
  2x -0:51.6.0 pane:show-next-item (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-light-syntax.theme-one-light-ui)
     -0:45 editor:cut-to-end-of-line (input.hidden-input)
     -0:45 pane:show-next-item (input.hidden-input)
     -0:41.6.0 core:save (input.hidden-input)
     -0:16.5.0 core:undo (div.console.repl-tooling.native-key-bindings)
     -0:14.9.0 core:redo (input.hidden-input)
     -0:08.4.0 core:save (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.33.4 
atomic-vale 1.9.0 
busy-signal 2.0.1 
chlorine 0.6.1 
color-picker 2.3.0 
compare-files 0.8.1 
copy-as-rtf 0.9.3 
dark-mode 4.0.2 
django-templates 0.5.0 
document-outline 2.1.2 
file-icons 2.1.40 
file-type-icons 1.3.4 
fold-lines 0.2.2 
fold-navigator 0.0.622 
graphviz-preview-plus 1.5.10 
highlight-line 0.12.0 
highlight-selected 0.17.0 
indent-guide-improved 1.4.13 
ink 0.12.3 
intentions 1.1.5 
language-cypher 0.5.0 
language-docker 1.1.8 
language-dot 1.7.0 
language-email 0.1.7 
language-eml 1.2.5 
language-ini 1.23.0 
language-kotlin 0.5.0 
language-markdown 0.37.0 
language-terraform 0.9.3 
linter 2.3.1 
linter-clojure 1.2.0 
linter-docker 0.3.2 
linter-eslint 8.5.5 
linter-flake8 2.4.0 
linter-joker 4.1.0 
linter-js-yaml 1.4.6 
linter-json-lint 0.1.3 
linter-jsonlint 1.4.0 
linter-kondo 1.2.0 
linter-markdown 5.2.11 
linter-markdownlint 2.0.2 
linter-remark 8.0.0 
linter-rubocop 2.5.7 
linter-shellcheck 1.6.0 
linter-ui-default 1.8.1 
markdown-folding 0.3.1 
markdown-writer 2.11.10 
mixed-indent-warning 0.3.0 
omni-ruler 0.6.0 
proto-repl 1.4.24 
rufo-atom 0.1.2 
sort-lines 0.19.0 
split-diff 1.6.1 
teletype 0.13.4 
title-case 0.4.0 
todo 0.6.0 
todo-show 2.3.2 
trailing-spaces 0.4.0 
tree-view-autoresize 2.3.0 
underline-trailing-whitespace 1.4.2 
wordcount 3.2.0 
Zen 0.18.0 
```

**Comment by mauricioszabo** [comment]
Hi, thanks for the report.

I assume that no REPL was connected when this error occurred, right?

**Comment by aviflax** [comment]
Sorry, I don’t really remember. I think there’s a pretty good chance that it was connected though.

**Comment by mauricioszabo** [comment]
@aviflax will be fixed on `0.6.1` (found a way to reproduce, the REPL indeed was not connected).

**Comment by aviflax** [comment]
Very nice!

**Comment by mauricioszabo** [comment]
Fixed on `0.6.2` (I forgot that 0.6.1 was already published :laughing:)
